### ISSUE TITLE
Error importing composer.lock

### ISSUE THREAD

**Issue by phackwer** [ludofleury/satisfy#60]
While trying to import a composer.lock, 

Fatal error: Call to a member function set() on null in /var/www/html/satisfy/src/Playbloom/Satisfy/Model/Manager.php on line 139

This could be easily avoided or better threated if you checked if 

```
        $this
        ->configuration
        ->getRepositories()
```

Returned something.

**Comment by phackwer** [comment]
There seem s to be a lot of  misthreated calls to methods on your code. I will fork it

**Comment by ramunasd** [comment]
Will be fixed completely with upcoming version 3.0
